build.gradle.kts

```diff
@@ -103,6 +90,8 @@ apply(from = "gradle/compatibility.gradle")
 
 plugins {
     id("org.jetbrains.dokka") version "1.6.10"
```

Comment:
you can create `[plugins]` config in version catalog TOML file: 
```
[plugins]
jmh = { id = "me.champeau.jmh", version = "0.6.5" }
```
and use it like this: `alias(libs.plugins.jmh)`
more info: https://docs.gradle.org/current/userguide/platforms.html#sec:plugins

Reply:
It also need changes in version injection for build train configuration, so we decided to postpone this